files/usr/lib/jeos-firstboot

```diff
@@ -266,45 +275,52 @@ fi
 default="$(readlink -f /etc/localtime)"
 default="${default##/usr/share/zoneinfo/}"
 
-# timedatectl doesn't work as dbus is not up yet
-# menulist timedatectl --no-pager list-timezones
-if menulist awk \
-    'BEGIN{print "UTC"; sort="sort"}/^#/{next;}{print $3|sort}END{close(sort)}' \
-    /usr/share/zoneinfo/zone.tab \
-    && d --default-item "$default" --menu $"Select Time Zone" 0 0 $dh_menu "${list[@]}"; then
+if [ -z "$JEOS_TIMEZONE" ]; then
+    # timedatectl doesn't work as dbus is not up yet
+    # menulist timedatectl --no-pager list-timezones
+    if menulist awk \
+		'BEGIN{print "UTC"; sort="sort"}/^#/{next;}{print $3|sort}END{close(sort)}' \
+		/usr/share/zoneinfo/zone.tab \
+	    && d --default-item "$default" --menu $"Select Time Zone" 0 0 $dh_menu "${list[@]}"; then
 	if [ -n "$result" ]; then
 	    systemd_firstboot_args+=("--timezone=$result")
 	fi
-else
+    else
 	d --msgbox $"error setting timezone" 0 0
+    fi
+else
+    timezone="$JEOS_TIMEZONE"
```

Comment:
`$timezone` can be replaced by `$JEOS_TIMEZONE` everywhere.

Reply:
Done